config/socket/socket.js

```diff
@@ -69,31 +69,42 @@ module.exports = function (io) {
         username: data
       };
       onlineUsers.push(user);
-      io.sockets.emit('people', onlineUsers)
-
+      io.sockets.emit('people', onlineUsers);
+      console.log('connect', onlineUsers);
```

Comment:
Please remove the console statement here

Reply:
I have done that, thanks